distributedlog-service/src/main/java/com/twitter/distributedlog/service/stream/StreamImpl.java

```diff
@@ -239,6 +231,12 @@ private DistributedLogManager openLog(String name) throws IOException {
     // Expensive initialization, only called once per stream.
     @Override
     public void initialize() throws IOException {
+        try {
+            sequencer = ReflectionUtils.newInstance(dynConf.getStreamSequencerClass());
```

Comment:
@leighst I was thinking of making this configured per stream. do you have any better suggestions on this?